bin/plotting/pycbc_page_snrratehist

```diff
@@ -165,7 +168,7 @@ if not args.closed_box:
             color='black',
             log=True,
             label='Full Background',
-            weights=dec / conv.sec_to_year(f.attrs['background_time_h%s' % h_inc_back_num])
+            weights=binweights
         )
```

Comment:
Huh.  This if-else switch seems to be redundant as the same hist call is made in both cases. 
Can the background attribute be defined in one-liner - eg `bg_key = 'background_time' if h_inc_back_num == 0 else 'background_time_h%s' % h_inc_back_num`
then the switch can be completely removed ?

Reply:
Added